### PR TITLE
MANY IMPROVEMENTS!

### PR DESCRIPTION
**Pull request by XFuryMCPE**
Okay, there are a few things this pull does:
- Max player count number is actually useful now
- New method for saving to server log (Possible file leak fix)
- New "Plus one max count" setting, set your slots to +1 the amount of players online
- Added startup script (Supports looping)
- Few minor changes to display and styling (just spacing errors)

Everything has been thoroughly tested and works perfectly. :)

### CODE REVIEW

**Comment by dktapps** [comment]
Please review the contribution guidelines before opening an pull request. Different repo, yes, but the same rules apply.

[Contribution guidelines](https://github.com/iTXTech/Genisys/blob/master/CONTRIBUTING.md)

**Comment by XFuryMCPE** [comment]
What the hell did I do wrong. There is no template for pulls

**Comment by legoboy0215** [comment]
@dktapps Why was this closed?

**Comment by PeratX** [comment]
Leave a comment

**Comment by PeratX** [comment]
@boybook Take a look at this!

**Comment by legoboy0215** [comment]
Thank you @PeratX.

**Comment by FilleDev** [comment]
Add this?

**Comment by dktapps** [comment]
@legoboy0215 It breaks several of the contribution guidelines, such as:
- 1 change per commit/PR
- Do not use GitHub Web because it's an unreliable piece of s***, and plenty of openings for inconsistencies
- Provide detailed explanations of changes

Also, XFury has proven to be a major irritation by failing to learn, and without fail every PR he has made to any iTX repo has had faults of some description because of improper testing and failure to follow the guidelines. 

Simply put, he has a bad reputation. (Some of those bad PRs were even merged by reviewers who didn't review properly, and caused issues for months afterwards.)

Large PRs are difficult to review, and especially unwelcome when coming from a dubious source such as this one.

Further details can be found [here](https://github.com/iTXTech/Genisys/blob/master/CONTRIBUTING.md).

One simple rule you should remember - if you want to contribute, you follow our rules.

**Comment by XFuryMCPE** [comment]
Please explain what I did that caused issues for months, lmfao I really soubt that, I've fixed a typo once and that got merged what else is there 😂😂

**Comment by legoboy0215** [comment]
Ah the sneaky @boybook XD

**Comment by dktapps** [comment]
@XFuryMCPE I don't suppose you remember a very poorly done PR to add installer options to Genisys?
Or the one to add unblock-address which was copy-pasted and caused server crashes until somebody else completed the job properly?

**Comment by dktapps** [comment]
@boybook on your head be it.

**Comment by XFuryMCPE** [comment]
Love you @boybook <3

**Comment by boybook** [comment]
I have read the files more than 5 times and I tested it.

**Comment by xZeroMCPE** [comment]
Lol I think @dktapps just wants fellow players to follow rules that's all 😂 Lol

**Comment by XFuryMCPE** [comment]
How about the rules get changed up then, some staff don't even mind, I don't see why @dktapps cares so much. If the code is bad, don't accept it. If the code is good, accept it. Stop worrying about the "commit count" or whatever, it doesn't make me a 'noob' to not know how to use git, because as you can see, I DO know how to code, and _most_ of my code is good with some exceptions I guess (i said most or dylan might've have a heart attack because he thinks my code is bad and I'm wrong). Back to the point, people should be able to edit however they want to get the work done, rather than worrying about the 'beauty' of the pull requests. You know how to use git? Sure, use it. You don't? That's okay, as long as the code works and is good. THAT's how I think it should be done. Sorry if you don't like my opinion, I don't really care, freedom of speech. :joy: 

Instead of checking if they used GitHub Web editor or not, check the code, it's more important.

Anyone else agree..?

**Comment by dktapps** [comment]
No professional will agree with you. And where one person fails, somebody else of better quality will step up to take their place.

**Comment by XFuryMCPE** [comment]
@PeratX and @boybook didn't seem to care how it was edited. They aren't professional ?

**Comment by dktapps** [comment]
Pfft. No offence to mentioned people, but no.

**Comment by xZeroMCPE** [comment]
@XFuryMCPE Just follow the contribution guidelines and such, contribution rules were created to keep things more organized...... Just get over it and follow it! :100:

**Comment by XFuryMCPE** [comment]
Wow, you're really cocky and self-centered @dktapps . You should take the opinions of other staff members of Genisys, because sorry to let you down, but you aren't the only one running this project. Seriously ask what ALL the staff members think about it and why, I think that'd be a good way of handling the issue, rather than saying everything I say is wrong. And I didn't ask for your negative comment @xZeroMCPE, your code is crap with or without git so you shouldn't be talking lmfao

**Comment by xZeroMCPE** [comment]
@XFuryMCPE At least i don't ask developers to code "gamemodes" for my own network. I can do it myself. lmao @XFuryMCPE  Your 100% crap + Immature.

**Comment by dktapps** [comment]
@XFuryMCPE You have a nerve slagging off other people's work when you simply refuse to learn. The honest truth is that I just plain don't like you, because you're a classic example of one of the idiots I deal with every day who refuse to use their brains.
